lib/std/crypto.zig

```diff
@@ -6,15 +6,14 @@
 
 /// Authenticated Encryption with Associated Data
 pub const aead = struct {
-    const chacha20 = @import("crypto/chacha20.zig");
-
     pub const Gimli = @import("crypto/gimli.zig").Aead;
-    pub const ChaCha20Poly1305 = chacha20.Chacha20Poly1305;
-    pub const XChaCha20Poly1305 = chacha20.XChacha20Poly1305;
+    pub const ChaCha20Poly1305 = @import("crypto/chacha20.zig").Chacha20Poly1305;
+    pub const XChaCha20Poly1305 = @import("crypto/chacha20.zig").XChacha20Poly1305;
     pub const Aegis128L = @import("crypto/aegis.zig").Aegis128L;
     pub const Aegis256 = @import("crypto/aegis.zig").Aegis256;
     pub const Aes128Gcm = @import("crypto/aes_gcm.zig").Aes128Gcm;
     pub const Aes256Gcm = @import("crypto/aes_gcm.zig").Aes256Gcm;
+    pub const XSalsa20Poly1305 = @import("crypto/salsa20.zig").XSalsa20Poly1305;
```

Comment:
What is the order of these? should they get sorted lexiographically?

Reply:
They are lexicographically sorted by core function name, and then by variants name. So that Salsa and XSalsa are paired together as well as ChaCha and XChaCha. It would be a little bit unnatural to spread them apart.